packages/mimecast/changelog.yml

```
@@ -1,3 +1,8 @@
+- version: "0.0.10"
+  changes:
+    - description: Add more use cases to audit-events pipeline, implent geo.ip for siem logs, remove user part for ttp-url logs and add email.to.address for recipient
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/3116
```

This could be split into a set of changes.

```suggestion
    - description: Add more use cases to audit-events pipeline
      type: enhancement
      link: https://github.com/elastic/integrations/pull/3116
    - description: Implement geo.ip for siem logs
      type: enhancement
      link: https://github.com/elastic/integrations/pull/3116
    - description: Remove user part for ttp-url logs and add email.to.address for recipient
      type: enhancement
      link: https://github.com/elastic/integrations/pull/3116
```